webpack.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  entry: './src/index.js',
+  module: {
+    rules: [
+      {
+        test: /\.(png|jp(e*)g|svg|gif)$/,
+        type: 'asset/resource',
+      },
+    ],
+  },
+};
```

Comment:
What is this for?

Reply:
webpack module is for implementing props.